cmd/main.go

```diff
@@ -39,11 +39,8 @@ func getConfigMetadata() map[string][]config.Parameter {
 			config.Parameter{Name: "logFile", Tag: "", Default: "/dev/stderr", Validators: []config.Validator{}},
 			config.Parameter{Name: "logLevel", Tag: "", Default: "INFO", Validators: []config.Validator{config.StringOptionsValidatorFactory([]string{"DEBUG", "INFO", "WARNING", "ERROR"})}},
 		},
-		"amqp1": []config.Parameter{
-			config.Parameter{Name: "connection", Tag: "", Default: "amqp://localhost:5672/lokean/logs", Validators: []config.Validator{}},
```

Comment:
Just changing default value to empty string and connect to amqp if there is any address would do ... same for the socket address. Theoretically single lokean could use both connection possibilities at once, receiving logs from different buses in paralel.

Reply:
Cool, I didn't think about this. I'll look into that tomorrow.